docs/src/pages/components/data-grid/editing/editing.md

```diff
@@ -87,7 +87,7 @@ Server-side validation works like client-side [validation](#validation).
 - Validate the value in the server.
 - Once the server responds, set the `error` attribute to false if it is valid. This allows to commit it.
 
-**Note:** To prevent the default client-side behavior, use `event.stopPropagation()`.
+**Note:** To prevent the default client-side behavior, set `event.defaultMuiPrevented` to `true`.
```

Comment:
if the user call `stopPropagation()`, should it also stop the propagation in MUI? 
If so we could override the function to something like
```ts
event.stopPropagation = ()=> {
  defaultMuiPrevented = true;
base.stopPropagation();
}
```
That would also keep the feature backward compatible.

Reply:
I think it's better to keep them independent, like calling `stopPropagation()` and `preventDefault()` in a JS event. This will be a real breaking change but we have to do it.